packages/xs-extra/xenctrl.master/opam

```diff
@@ -8,18 +8,13 @@ bug-reports: "https://github.com/xapi-project/xenctrl/issues"
 depends: [
   "ocaml"
   "dune" {build}
+  "conf-m4"
```

Comment:
Does it really need m4?

Reply:
No, although we're not really using this library, we're using the copy in Xen.